.bazelrc

```diff
@@ -21,7 +21,7 @@ build --experimental_remote_cache_eviction_retries=3
 # due to https://github.com/bazelbuild/bazel/issues/20576.
 # Windows build is failing on some runs due to FileAccessException errors, Bazel has had several
 # similar bugs in the past. Disable BwoB for now.
-build:windows --remote_download_all
+build:windows --remote_download_outputs=all
```

Comment:
Non-blocking: This overlaps with #2981, where we should be able to get rid of this again